code_schemes/constituency.json

```diff
@@ -0,0 +1,214 @@
+{
+    "SchemeID": "Scheme-4c5b955d",
+    "Name": "Constituency",
+    "Version": "0.0.0.1",
+    "Codes": [
+      {
+        "CodeID": "code-bd610414",
+        "CodeType": "Normal",
+        "DisplayText": "Mwea",
+        "StringValue": "mwea",
+        "NumericValue": 1,
+        "VisibleInCoda": true,
+        "MatchValues": [
+          "mwea"
+        ]
+      },
+      {
+        "CodeID": "code-4e32405d",
+        "CodeType": "Normal",
+        "DisplayText": "Gichugu",
+        "StringValue": "gichugu",
+        "NumericValue": 2,
+        "VisibleInCoda": true,
+        "MatchValues": [
+          "gichugu"
+        ]
+      },
+      {
+        "CodeID": "code-963ec546",
+        "CodeType": "Normal",
+        "DisplayText": "Ndia",
+        "StringValue": "ndia",
+        "NumericValue": 3,
+        "VisibleInCoda": true,
+        "MatchValues": [
+          "ndia"
+        ]
+      },
+      {
+        "CodeID": "code-c7ec0beb",
+        "CodeType": "Normal",
+        "DisplayText": "Tetu",
+        "StringValue": "tetu",
+        "NumericValue": 3,
```

Comment:
These shouldn't have the same numeric value

Reply:
Thanks @lukechurch have updated!